docs/coreConcepts/LitActionsAndPKPs/litActions.md

```diff
@@ -24,6 +24,22 @@ Lit Actions are essentially decentralized serverless functions. You can use Lit
 
 A toy example would be a Lit Action and corresponding PKP that checks if a number is prime, and only signs it if it is prime.  Think of it as a sort of “prime number” certification service.  Since the Lit Action is immutable, and since you can permanently assign a PKP to a Lit Action, there is a provable chain of trust.  This means you could present the signature and a number to someone, and they could simply check the signature against the public key of the PKP to see if the number is actually prime, instead of having to do all the math to ensure that the number is actually prime.  The signature acts as a proof that the number is prime.  
 
+## Signatures vs. Proofs
+
+It is important to breakdown the distinction between a "signature" and what can be considered a "proof".
+
+### Signing
+
+In cryptography, a digital signature proves that a user controls the private key associated with a particular public address. This is commonly used in the context of *authorization*. Apps can use this signature to verify a user has control over the digital identity provided by the address, and provide authentication to do both web3 and arbitrary, non-web3 actions. For example, changing a profile on OpenSea requires signing in order to confirm the association between the profile owner the the wallet associated with it.
+
+In working with the Lit network, someone will sign with their wallet when they want to mint a PKP. That is a transaction signature — where someone is sending a transaction (minting). This signature is than used as the method of authentication over the PKP and associated Lit Actions.
+
+### Proofs
+
+A proof is a particular application for a digitial signature. For example, using a signature to *prove* that a particular interaction took place. 
+
+Signing through Lit Actions opens up the possibilities of verifying information from external sources, such as a Weather API. Within the Lit ecosystem, this is considered a proof. The proof sends back information from a source such as endpoint or function, which you can then harness within your own application. 
```

Comment:
Paragraph is a bit redundant, rewording it

"Signing through Lit Actions opens up the possibilities of verifying information from external sources, such as from the Weather API. Within the Lit ecosystem this signed information is called a proof. "